apps/web/lib/hooks/useIsBookingPage.ts

```diff
@@ -4,7 +4,9 @@ import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
 
 export default function useIsBookingPage(): boolean {
   const pathname = usePathname();
-  const isBookingPage = ["/booking/", "/cancel", "/reschedule"].some((route) => pathname?.startsWith(route));
+  const isBookingPage = ["/booking/", "/cancel", "/reschedule", "/apps/routing-forms/routing-link"].some(
+    (route) => pathname?.startsWith(route)
+  );
```

Comment:
correct me if I'm wrong:

- /apps/routing-forms/routing-link/6aae08e3-bb0b-4fd3-9f9b-00325191c1e9
- /forms/6aae08e3-bb0b-4fd3-9f9b-00325191c1e9

are these the same? If so, should we include "/forms/" here?

Reply:
Great question. Asked Hariom and here's what he said:
> Yeah, they are same. /forms/{FORM_ID} is how users access that page as that is a public facing link

Reply:
Oh I see

```
      {
         source: "/forms/:formQuery*",
         destination: "/apps/routing-forms/routing-link/:formQuery*",
       }
```

then do we want to include /forms/ here, or not?

Reply:
Yeah we should